tests/legacy/system/test_edb.py

```diff
@@ -1732,3 +1730,94 @@ def test_multizone(self, edb_examples):
         assert defined_ports
         assert project_connexions
         edbapp.close_edb()
+
+    @pytest.mark.skipif(
+        not desktop_version == "2024.2" or int(desktop_version.split(".")[0]) >= 2025,
+        reason="Only supported with 2024.2 and higher",
+    )
+    def test_add_raptorx_setup(self):
+        source_path = os.path.join(local_path, "example_models", test_subfolder, "ANSYS-HSD_V1.aedb")
+        target_path = os.path.join(self.local_scratch.path, "test_raptorx_setup", "test.aedb")
+        self.local_scratch.copyfolder(source_path, target_path)
+        edbapp = Edb(edbpath=target_path, edbversion=desktop_version)
+        setup = edbapp.create_raptorx_setup("test")
+        assert "test" in edbapp.setups
+        setup.add_frequency_sweep(frequency_sweep=["linear scale", "0.1GHz", "10GHz", "0.1GHz"])
+        setup.enabled = False
+        assert not setup.enabled
+        assert len(setup.frequency_sweeps) == 1
+        general_settings = setup.settings.general_settings
+        assert general_settings.global_temperature == 22.0
```

Comment:
Can you change all property assertion as below? This makes sure all changes are written into edb.
assert edbapp.setups["test"].settings.general_settings-global_temperature == 22.0